src/chrome/content/rules/LongTail.xml

```diff
@@ -1,67 +1,89 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://content.longtailvideo.com/ => https://content.longtailvideo.com/: (7, 'Failed to connect to content.longtailvideo.com port 443: Connection timed out')
-Fetch error: http://dashboard.longtailvideo.com/ => https://dashboard.longtailvideo.com/: (28, 'Connection timed out after 20000 milliseconds')
-Fetch error: http://secure.longtailvideo.com/ => https://secure.longtailvideo.com/: (28, 'Connection timed out after 20005 milliseconds')
-Fetch error: http://developer.longtailvideo.com/ => https://developer.longtailvideo.com/: (51, "SSL: no alternative certificate subject name matches target host name 'developer.longtailvideo.com'")
-
 	Other LongTail rulesets:
 		- Bitsontherun.com.xml
 		- JW_Platform.com.xml
 
-	Valid, problematic (March 2016):
-		<target host="ssl.p.jwpcdn.com" />(breaks video plugins, e.g. pbs.org)
-
-	Mismatch/NXDOMAIN (March 2016):
-		<target host="cdn.s.jwpcdn.com" />
-		<target host="a.jwpcdn.com" />
-		<target host="v.jwpcdn.com" />
-		<target host="p.jwpcdn.com" />
-		<target host="i.jwpltx.com" />
-
-		<target host="player.longtailvideo.com" />
-
-		<target host="ltvimg.com" />
-		<target host="s0-www.ltvimg.com" />
-		<target host="s1-www.ltvimg.com" />
-
-		<target host="insights.jwplayer.com" />
-		<target host="status.jwplayer.com" />
-
-	Timeout/Refused (March 2016):
-		<target host="jw7.jwplayer.com" />
-		<target host="demo.jwplayer.com" />
-		<target host="sdk.jwplayer.com" />
```

Comment:
Please keep this host around. LGTM apart from that.
Please don't overwrite this commit in your branch when fixing this.

Reply:
ping @numismatika

Reply:
sdk got lost in the momentum. Still doesn't respond unfortunately